modules/ROOT/pages/import-api-specification-design-center.adoc

```diff
@@ -0,0 +1,44 @@
+= Importing an API Specification from Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API Definition from Design Center into either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
```

Comment:
I suggest moving this point into the intro, by specifying RAML there (if we mean the same thing by "specification" as by "definition":

"Anypoint Studio enables you to import a RAML API definition from Design Center into either a new Mule project or an existing one."

Perhaps we could add the size limitation here as well:

"Anypoint Studio enables you to import a RAML API definition of fewer than 4,000 lines from Design Center into either a new Mule project or an existing one."

Just ideas....